src/cdi_analytic/Analytic_CP_Eloss.cc

```diff
@@ -30,15 +30,15 @@ namespace rtt_cdi_analytic {
  * \param projectile_zaid_in int32_t transporting particle zaid
  */
 Analytic_CP_Eloss::Analytic_CP_Eloss(SP_Analytic_Model model_in,
-                                     int32_t target_zaid_in,
-                                     int32_t projectile_zaid_in)
-    : analytic_model(model_in), projectile_zaid(projectile_zaid_in),
-      target_zaid(target_zaid_in) {
+                                     rtt_cdi::CParticle &target_in,
+                                     rtt_cdi::CParticle &projectile_in)
+    : analytic_model(model_in), rtt_cdi::CPEloss(target_in, projectile_in) {
```

Comment:
Same here, I think the base class constructor call needs to be called before analytic_model is set :)